### ISSUE TITLE
Weekly Notifications

### ISSUE THREAD

**Issue by uBadRequest** [d3vzer0/streamio#4]
Hey!

Nice project, i'm interesting in using this to monitor for trademark infringement. I'm looking to push notifications to a Microsoft Teams channel every week with each new domain that has been found in a message with a screenshot. 

I could add the integration with Teams myself, as I don't expect you to specifically want to add Microsoft Teams, maybe Discord, but is there currently any methods for sending notifications? I couldn't find anything when looking/searching through the code. I haven't installed this yet on a server, but might look into it tomorrow. 

I was thinking about just querying this source weekly: https://tools.whoisxmlapi.com/domain-subdomain-discovery
I also have access to https://czds.icann.org and have been thinking about how I could use https://opendata.rapid7.com but I don't really want to download all the data(its like 20 or 30GB) every week just to search for a keyword.

Would you consider adding any of these data sources I mentioned?

**Comment by blueteamzone** [comment]
Hey @uBadRequest did you manage to install the repo without issues?

**Comment by uBadRequest** [comment]
> Hey @uBadRequest did you manage to install the repo without issues?

Did you clone the repo recursively? If not, you would have missed the submodules. Or are you trying to set this up locally? In that case you need to edit the playbook so it runs everything locally and not try to use ssh.

**Comment by blueteamzone** [comment]
@uBadRequest Yeah, we cloned the repo recursively and we are trying to set it up running all `locally.` 
What are the changes that we have to make to the playbooks?

The only modification we made was to the following file:
`inventories/development/host_vars/phish-dev-01`
changing the IP address to the local IP address
